jerry-debugger/jerry_client_main.py

```diff
@@ -418,20 +422,19 @@ def delete(self, args):
             return "Error: Integer number expected, %s\n" % (val_errno)
 
         if breakpoint_index in self.active_breakpoint_list:
-            breakpoint = self.active_breakpoint_list[breakpoint_index]
+            br_point = self.active_breakpoint_list[breakpoint_index]
             del self.active_breakpoint_list[breakpoint_index]
-            breakpoint.active_index = -1
-            self._send_breakpoint(breakpoint)
+            br_point.active_index = -1
+            self._send_breakpoint(br_point)
             return "Breakpoint %d deleted\n" % (breakpoint_index)
-        elif breakpoint_index in self.pending_breakpoint_list:
+        if breakpoint_index in self.pending_breakpoint_list:
             del self.pending_breakpoint_list[breakpoint_index]
             if not self.pending_breakpoint_list:
                 self._send_parser_config(0)
             return "Pending breakpoint %d deleted\n" % (breakpoint_index)
-        else:
-            return "Error: Breakpoint %d not found\n" % (breakpoint_index)
+        return "Error: Breakpoint %d not found\n" % (breakpoint_index)
 
-    def next(self):
+    def __next__(self):
```

Comment:
This should not be changed. The `__next__` would mean that this is an iterator/iterable, but this is not an iterator but a command sent via the debugger connection.